lib/susedistribution.pm

```diff
@@ -232,10 +232,7 @@ sub ensure_installed {
     testapi::assert_script_run('systemctl is-active -q packagekit || (systemctl unmask -q packagekit ; systemctl start -q packagekit)');
     type_string "exit\n";
     my $retries = 5;    # arbitrary
-    $self->script_run(
-"for i in {1..$retries} ; do pkcon install -yp $pkglist && break ; done ; RET=\$?; echo \"\n  pkcon finished\n\"; echo \"pkcon-\${RET}-\" > /dev/$testapi::serialdev",
-        0
-    );
+    $self->script_run("pkcon install -yp $pkglist; echo \"pkcon-\$?-\" > /dev/$testapi::serialdev", 0);
```

Comment:
I guess we don't need extra exit code output if we don't have a loop, script_run already does it.

Reply:
no, I don't think so. Compare to https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/tests/console/yast2_apparmor.pm#L26

Reply:
Ah, I see, we do some magic in between.